core/src/main/java/com/linecorp/armeria/client/HttpRequestSubscriber.java

```diff
@@ -73,6 +73,7 @@
     @Nullable
     private ScheduledFuture<?> timeoutFuture;
     private State state = State.NEEDS_TO_WRITE_FIRST_HEADER;
+    private boolean isCompleted;
```

Comment:
How about renaming something like `isSubscriptionCompeted`?
I think this makes me confused as it's hard to figure out the difference with `isDone()`.

Reply:
It makes sense. It can be confused because it's too general. Thanks!